doc/api/modules.md

```diff
@@ -552,7 +552,18 @@ added: v0.1.13
 
 * {Function}
 
-To require modules.
+Used to import modules, `JSON` and local files. Modules can be imported from `node_modules`. Local modules and JSON files can be imported using the pattern `'./`.
```

Comment:
is there a single quote missing between the back-quotes?

Reply:
fixed! :D